### ISSUE TITLE
The focus rectangles are really annoying and look bad

### ISSUE THREAD

**Issue by QuietRobot** [microsoft/microsoft-ui-xaml#5633]
This is a problem that exists since Windows 10, and now with windows 11 it seems that it is worse.
Constantly when you interact with any element of the interface, a white box (dark theme) or black (light theme) appears... 
**shouldn't this happen only when using "tab"?** 

thanks.

examples: [https://i.imgur.com/oZiQOK9.png](url)

**Comment by fitojb** [comment]
I’m OK with those, actually, but maybe having them take the accent color will make them more palatable for more people.

**Comment by hyperactiverobot** [comment]
> 
> 
> I’m OK with those, actually, but maybe having them take the accent color will make them more palatable for more people.

what do you need to see them for?

**Comment by mdtauk** [comment]
They exist for keyboarding, but occasionally show up when only using mouse inputs.

You would use Tab and Arrow Keys to highlight controls showing what you are interacting with, and they are used by accessibility tools and screen readers to highlight the currently focused or selected control

**Comment by hyperactiverobot** [comment]
> 
> 
> They exist for keyboarding, but occasionally show up when only using mouse inputs.
> 
> You would use Tab and Arrow Keys to highlight controls showing what you are interacting with, and they are used by accessibility tools and screen readers to highlight the currently focused or selected control

Exactly! that's what I mean, they should only be seen when using "tab". I hope it will be fixed soon.

**Comment by StephenLPeters** [comment]
tracking this internally [here](https://microsoft.visualstudio.com/OS/_workitems/edit/34784936)

**Comment by hyperactiverobot** [comment]
> 
> 
> Thanks @hyperactiverobot I believe this is a known issue we are still looking at. Can you give us more information about how you found the issue to make sure?
> 
> tracking this internally [here](https://microsoft.visualstudio.com/OS/_workitems/edit/34784936)

That's great. Generally it is random, but there are interfaces that are almost 100% sure to appear, for example the taskbar language and action center flyout, or the popup messages.

**Comment by kmgallahan** [comment]
@StephenLPeters In my experience you usually get a focus box any time you launch an app **for the first time** by hitting the windows key + typing in some part of the app name and hitting enter.

After the first time it seems to happen less, but somewhat unpredictably.

**Comment by StephenLPeters** [comment]
Great, sounds the same.

**Comment by hyperactiverobot** [comment]
@StephenLPeters this is another interface where it happens almost 100% of the time: https://i.imgur.com/Ln7ZWDP.png

**Comment by niels9001** [comment]
From #9683: 

<img width="627" height="707" alt="Image" src="https://github.com/user-attachments/assets/6b62e96c-da09-4ef2-8ce6-6d867ffd4a66" />

**Comment by mdtauk** [comment]
Ideally no focus rectangle should be visible until the Tab key has been pressed.

**Comment by QuietRobot** [comment]
@mdtauk 25H2 26200.8655
The same happens when opening some inboxes and third-party apps, or some sections of Settings.

https://github.com/user-attachments/assets/a461f921-e5b8-42e0-906b-a138b709d757
